client/MANIFEST.in

```diff
@@ -0,0 +1 @@
+include src/ledger_app_clients/ethereum/keychain/*
```

Comment:
newline at end of file

Reply:
Also what does this line mean and what does this file do ?

Reply:
This is needed to include the text files stored into this path into the Python package, else only `.py` file would be included.
This files are needed by the `keychain,py` module.